### DISCUSSION TITLE
Zed for blazor projects?

### DISCUSSION THREAD

**Discussion by marc-hg** [zed-industries/zed#8332] [Feature Requests]
I just tried opening a blazor project on zed but I don't see any LSP actions nor syntax highlighting

**Comment by Moshyfawn** [comment]
At present, Zed supports C# `.cs` files but not `.razor` files.

**Reply by csteenbock** [reply]
Are there any plans to support `.razor` files?

**Comment by michalss** [comment]
This would be perfect to support razor files..
